src/installation/index.md

```diff
@@ -1,160 +1,203 @@
 
-Regardless of the target architecture, if you want to build a project using the [`std` approach], you will also need:
-- [ESP-IDF]: Espressif IoT Development Framework as it's used as our hosted environment.
-  - This is handled by [`esp-idf-sys`] (a crate that all `std` projects need to use) by default. See [ESP-IDF installation note] for details on how ESP-IDF can be installed.
-- [`ldproxy`] crate:  Simple tool to forward linker arguments given to [`ldproxy`] to the actual linker executable. The crate can be found in the [esp-rs/embuild] repository. To install it, use the following command:
+
+[llvm-github-fork]: https://github.com/espressif/llvm-project
+[gcc-toolchain-github-fork]: https://github.com/espressif/crosstool-NG/
+[rustup-overrides]: https://rust-lang.github.io/rustup/overrides.html
+[llvm-github-fork-upstream issue]: https://github.com/espressif/llvm-project/issues/4
+
+
+## `std` Development Requirements
+
+Regardless of the target architecture, make sure you have the following required tools installed to build [`std`][rust-esp-book-overview-std] applications:
+
+- [`python`][python-website-download]: Required by ESP-IDF
+- [`git`][git-website-download]: Required by ESP-IDF
+- [`ldproxy`][embuild-github-ldproxy] crate: Simple tool to pass the `ldproxy` linker arguments to the actual linker executable. Install it  by running
   - `cargo install ldproxy`
 
+The std runtime uses [ESP-IDF][esp-idf-github] (Espressif IoT Development Framework) as hosted environment but, users do not need to install it. ESP-IDF is automatically downloaded and installed by [esp-idf-sys][esp-idf-sys-github], a crate that all std projects need to use, when building a std application.
+
+
+[rust-esp-book-overview-std]: ../overview/using-the-standard-library.md
+[python-website-download]: https://www.python.org/downloads/
+[git-website-download]: https://git-scm.com/downloads
+[embuild-github-ldproxy]: https://github.com/esp-rs/embuild/tree/master/ldproxy
+[esp-idf-sys-github]: https://github.com/esp-rs/esp-idf-sys
+[esp-idf-github]: https://github.com/espressif/esp-idf
 
-[ESP-IDF]: https://github.com/espressif/esp-idf
-[`std` approach]: ../overview/using-the-standard-library.md
-[`ldproxy`]: https://github.com/esp-rs/embuild/tree/master/ldproxy
-[esp-rs/embuild]: https://github.com/esp-rs/embuild
-[ESP-IDF installation note]: https://github.com/esp-rs/espup#esp-idf-instalation
 
 ## Using Containers
 
-As an alternative to installing the environment directly on your local system, it's also possible to run it inside a container.
+Instead of installing directly on your local system, you can host the development environment inside a container. Espressif provides the [idf-rust] image that supports both `RISC-V` and `Xtensa` target architectures and enables both `std` and `no_std` development.
+
+You can find numerous tags for `linux/arm64` and `linux/amd64` platforms.
```

Comment:
```suggestion
You can find numerous tags for `linux/arm64`, and `linux/amd64` platforms.
```

Reply:
I am afraid the comma is not needed here in English. See the details [here](https://www.scribbr.com/commas/comma-before-or-after-and/).